client/lib/safe-image-url/test/index.js

```diff
@@ -1,68 +1,78 @@
-		expect( safeImage( 'http://wordpress.com/' ) ).to.eql( 'https://wordpress.com/' );
+	it( 'should promote an http wpcom url to https', () => {
+		expect( safeImageUrl( 'http://files.wordpress.com/' ) ).to.equal( 'https://files.wordpress.com/' );
+		expect( safeImageUrl( 'http://wordpress.com/' ) ).to.equal( 'https://wordpress.com/' );
 	} );
 
-	it( 'should leave https wpcom url alone', function() {
-		expect( safeImage( 'https://files.wordpress.com/' ) ).to.eql( 'https://files.wordpress.com/' );
-		expect( safeImage( 'https://wordpress.com/' ) ).to.eql( 'https://wordpress.com/' );
-		expect( safeImage( 'https://blog-en.files.wordpress.com/' ) ).to.eql( 'https://blog-en.files.wordpress.com/' );
+	it( 'should leave https wpcom url alone', () => {
+		expect( safeImageUrl( 'https://files.wordpress.com/' ) ).to.equal( 'https://files.wordpress.com/' );
+		expect( safeImageUrl( 'https://wordpress.com/' ) ).to.equal( 'https://wordpress.com/' );
+		expect( safeImageUrl( 'https://blog-en.files.wordpress.com/' ) ).to.equal( 'https://blog-en.files.wordpress.com/' );
 	} );
 
-	it( 'should promote an http gravatar url to https', function() {
-		expect( safeImage( 'http://files.gravatar.com/' ) ).to.eql( 'https://files.gravatar.com/' );
-		expect( safeImage( 'http://gravatar.com/' ) ).to.eql( 'https://gravatar.com/' );
+	it( 'should promote an http gravatar url to https', () => {
+		expect( safeImageUrl( 'http://files.gravatar.com/' ) ).to.equal( 'https://files.gravatar.com/' );
+		expect( safeImageUrl( 'http://gravatar.com/' ) ).to.equal( 'https://gravatar.com/' );
 	} );
 
-	it( 'should leave https gravatar url alone', function() {
-		expect( safeImage( 'https://files.gravatar.com/' ) ).to.eql( 'https://files.gravatar.com/' );
-		expect( safeImage( 'https://gravatar.com/' ) ).to.eql( 'https://gravatar.com/' );
+	it( 'should leave https gravatar url alone', () => {
+		expect( safeImageUrl( 'https://files.gravatar.com/' ) ).to.equal( 'https://files.gravatar.com/' );
+		expect( safeImageUrl( 'https://gravatar.com/' ) ).to.equal( 'https://gravatar.com/' );
 	} );
 
-	it( 'should return null for urls with querystrings', function() {
-		expect( safeImage( 'https://example.com/foo?bar' ) ).to.be.null;
-		expect( safeImage( 'https://example.com/foo.jpg?bar' ) ).to.be.null;
-		expect( safeImage( 'https://example.com/foo.jpeg?bar' ) ).to.be.null;
-		expect( safeImage( 'https://example.com/foo.gif?bar' ) ).to.be.null;
-		expect( safeImage( 'https://example.com/foo.png?bar' ) ).to.be.null;
+	it( 'should return null for urls with querystrings', () => {
+		expect( safeImageUrl( 'https://example.com/foo?bar' ) ).to.be.null;
+		expect( safeImageUrl( 'https://example.com/foo.jpg?bar' ) ).to.be.null;
+		expect( safeImageUrl( 'https://example.com/foo.jpeg?bar' ) ).to.be.null;
+		expect( safeImageUrl( 'https://example.com/foo.gif?bar' ) ).to.be.null;
+		expect( safeImageUrl( 'https://example.com/foo.png?bar' ) ).to.be.null;
```

Comment:
we should probably add a few tests with weird protocols too, especially `javascript:`, maybe `about:` and `config:`? Doesn't have to be this PR.